_includes/workshops/cvpr/cvpr2021.html

```diff
@@ -91,9 +97,10 @@ <h2 class="blog-title">Extended Abstracts</h2>
           <col style="width: 5%">
           <col style="width: 35%">
```

Comment:
@anthxnyR the title column width should be increased more so that the category tabs remain on one line and are not stretched.

Reply:
You may need to override the theme styling to increase the overall table width to make room for extending the columns.

Reply:
Try overriding the max-width setting for the main class.
![Screen Shot 2021-09-11 at 2 31 00 PM](https://user-images.githubusercontent.com/14129918/132959437-f3dd27c0-34c4-47e6-998d-faba923bba79.png)